include/sequence.h

```diff
@@ -160,119 +161,120 @@ typedef enum {
+    /* 0x0B */ AMBIENCE_ID_0B,
+    /* 0x0C */ AMBIENCE_ID_0C,
+    /* 0x0D */ AMBIENCE_ID_0D,
+    /* 0x0E */ AMBIENCE_ID_0E,
+    /* 0x0F */ AMBIENCE_ID_0F,
+    /* 0x10 */ AMBIENCE_ID_10,
+    /* 0x11 */ AMBIENCE_ID_11,
+    /* 0x12 */ AMBIENCE_ID_12,
+    /* 0x13 */ AMBIENCE_ID_13,
+    /* 0xFF */ AMBIENCE_ID_DISABLED = 0xFF
+} AmbienceId; // playerIndex = 4
 
 typedef enum {
-    /* 0x00 */ NATURE_STREAM_0,
-    /* 0x01 */ NATURE_STREAM_1,
-    /* 0x02 */ NATURE_STREAM_2,
-    /* 0x03 */ NATURE_STREAM_3
-} NatureStreamId;
+    /* 0x00 */ AMBIENCE_STREAM_0,
+    /* 0x01 */ AMBIENCE_STREAM_1,
+    /* 0x02 */ AMBIENCE_STREAM_2,
+    /* 0x03 */ AMBIENCE_STREAM_3
+} AmbienceStreamId;
 
 typedef enum {
-    /* 0x00 */ NATURE_CRITTER_00,
-    /* 0x01 */ NATURE_CRITTER_01,
-    /* 0x02 */ NATURE_CRITTER_02,
-    /* 0x03 */ NATURE_CRITTER_03,
-    /* 0x04 */ NATURE_CRITTER_04,
-    /* 0x05 */ NATURE_CRITTER_05,
-    /* 0x06 */ NATURE_CRITTER_06,
-    /* 0x07 */ NATURE_CRITTER_07,
-    /* 0x08 */ NATURE_CRITTER_08,
-    /* 0x09 */ NATURE_CRITTER_09,
-    /* 0x0A */ NATURE_CRITTER_10,
-    /* 0x0B */ NATURE_CRITTER_11,
-    /* 0x0C */ NATURE_CRITTER_12,
-    /* 0x0D */ NATURE_CRITTER_13,
-    /* 0x0E */ NATURE_CRITTER_14,
-    /* 0x0F */ NATURE_CRITTER_15,
-    /* 0x10 */ NATURE_CRITTER_16,
-    /* 0x11 */ NATURE_CRITTER_17,
-    /* 0x12 */ NATURE_CRITTER_18,
-    /* 0x13 */ NATURE_CRITTER_19
-} NatureAmimalId;
+    /* 0x00 */ AMBIENCE_CRITTER_00,
+    /* 0x01 */ AMBIENCE_CRITTER_01,
+    /* 0x02 */ AMBIENCE_CRITTER_02,
+    /* 0x03 */ AMBIENCE_CRITTER_03,
+    /* 0x04 */ AMBIENCE_CRITTER_04,
+    /* 0x05 */ AMBIENCE_CRITTER_05,
+    /* 0x06 */ AMBIENCE_CRITTER_06,
+    /* 0x07 */ AMBIENCE_CRITTER_07,
+    /* 0x08 */ AMBIENCE_CRITTER_08,
+    /* 0x09 */ AMBIENCE_CRITTER_09,
+    /* 0x0A */ AMBIENCE_CRITTER_10,
+    /* 0x0B */ AMBIENCE_CRITTER_11,
+    /* 0x0C */ AMBIENCE_CRITTER_12,
+    /* 0x0D */ AMBIENCE_CRITTER_13,
+    /* 0x0E */ AMBIENCE_CRITTER_14,
+    /* 0x0F */ AMBIENCE_CRITTER_15,
+    /* 0x10 */ AMBIENCE_CRITTER_16,
+    /* 0x11 */ AMBIENCE_CRITTER_17,
+    /* 0x12 */ AMBIENCE_CRITTER_18,
+    /* 0x13 */ AMBIENCE_CRITTER_19
+} AmbienceCritterId;
 
-#define NATURE_IO_CRITTER_0_TYPE(type)        NATURE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_2, type
-#define NATURE_IO_CRITTER_0_BEND_PITCH(bend)  NATURE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_3, bend
-#define NATURE_IO_CRITTER_0_NUM_LAYERS(num)   NATURE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_4, num
-#define NATURE_IO_CRITTER_0_PORT5(reverb)     NATURE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_5, reverb
+#define AMBIENCE_IO_CRITTER_0_TYPE(type)        AMBIENCE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_2, type
+#define AMBIENCE_IO_CRITTER_0_BEND_PITCH(bend)  AMBIENCE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_3, bend
+#define AMBIENCE_IO_CRITTER_0_NUM_LAYERS(num)   AMBIENCE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_4, num
+#define AMBIENCE_IO_CRITTER_0_PORT5(reverb)     AMBIENCE_CHANNEL_CRITTER_0, CHANNEL_IO_PORT_5, reverb
```

Comment:
Is there a reason for having a different set of macros for every sort of ambience?

Reply:
I found
`AMBIENCE_IO_CRITTER_0_TYPE(AMBIENCE_CRITTER_09)`
to be better than:
`AMBIENCE_IO_TYPE(AMBIENCE_CHANNEL_CRITTER_0, AMBIENCE_CRITTER_09)`

Plus, ioPorts 3/4/5 have different functions for critters/streams, so the macro wouldn't even work for those ioPorts